packages/x-adapter-platform/src/mappers/requests/tagging-request.mapper.ts

```diff
@@ -1,7 +1,7 @@
-import { schemaMapperFactory } from '@empathyco/x-adapter';
+import { Mapper } from '@empathyco/x-adapter';
 import { TaggingRequest } from '@empathyco/x-types';
-import { taggingRequestMutableSchema } from '../../schemas/requests/tagging-request.schema';
 
-export const taggingRequestMapper = schemaMapperFactory<TaggingRequest, any>(
-  taggingRequestMutableSchema
-);
+export const taggingRequestMapper: Mapper<TaggingRequest, any> = ({
```

Comment:
Is the wrapping of the params object intentional?

Also, due to this change, the `tagging-request-mapper.spec.ts` test fails:

```ts
 it('should map the request', () => {
    const internalRequest: TaggingRequest = {
      url: 'https://api.staging.empathy.co/tagging/v1/track/empathy/click',
      params: {
        filtered: 'false',
        follow: false,
        lang: 'en',
        origin: 'search_box:none',
        page: '1',
        position: '1',
        productId: '12345-U',
        q: '12345',
        scope: 'desktop',
        spellcheck: 'false',
        title: 'Xoxo Women Maroon Pure Georgette Solid Ready-to-wear Saree'
      }
    };

    expect(taggingRequestMapper(internalRequest, {})).toStrictEqual({
      params: {
        filtered: 'false',
        follow: false,
        lang: 'en',
        origin: 'search_box:none',
        page: '1',
        position: '1',
        productId: '12345-U',
        q: '12345',
        scope: 'desktop',
        spellcheck: 'false',
        title: 'Xoxo Women Maroon Pure Georgette Solid Ready-to-wear Saree'
      }
    });
```

Reply:
no, it was not intended, my fault.

Fixed now